File: python/completed/euler0019.py
###
#
# Created by Holden on 12/18/2015
#
# SOLVED on 12/18/2015
#
# Problem 19 - Counting Sundays
#
###

import datetime

def counting_sundays(start, end):
    count = 0
    
    for year in range(start, end + 1):
        for month in range(1, 13):
            date = datetime.date(year, month, 1)
            if date.weekday() == 6:
                count += 1
                
    return count

File: javascript/completed/euler0019.js
export async function solve19() {
  const day = 6; // Monday
  let total = 0;
  for (var year = 1901; year <= 2000; year++) {
    for (var month = 1; month <= 12; month++) {
      total += new Date(`${month}/1/${year}`).getDay() === day ? 1 : 0;
    }
  }
  return total;
}
